client/app/components/packages/pas-form/edit.hbs

```diff
@@ -826,7 +826,7 @@
                 <button
                   type="button"
                   class="button large no-margin"
-                  {{on "click" (fn this.submit @package)}}
+                  onClick={{fn this.submit @package}}
```

Comment:
What's wrong with the modifier?

Reply:
Oh nothing's wrong, was just updating for consistency -- I can change back if you prefer the other way

Reply:
Hmm consistent with what? The click handling above uses modifiers.

I'm not sure what's best but onClick/onclick has confused me because onclick is a native event callback. onClick seems to work too, and I don't know why, because it's camel-case. 

I think the modifier is maybe more explicit, it's clearly part of Ember and not ambiguously 'native html or some older maybe-deprecated ember api'... what do you think?